ansible/roles/elastic_beats/vars/tenderdash.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+platform_filebeat_inputs:
```

Comment:
If we name the top level of the filebeat templates like `core_filebeat_inputs`, `tenderdash_filebeat_inputs`, etc., it will result in more conditional logic to handle seed nodes because you cannot merge with an undefined variable, and we cannot initialize variables due to precedence rules. So we would either need an extra step to rename the top level of `tenderdash_filebeat_inputs`, or three final merge steps, depending on if we are running on a masternode, seed node or evonode.

For this reason I think it is less confusing to give the top level of the partial filebeat template we are merging into a predictable name that we can then reference in following steps, eventually merging everything into `filebeat_inputs`, which is directly ready for consumption by the `geerlingguy.filebeat` role.

I remember struggling with this a lot when I first wrote the templates.